db/change_cache.go

```diff
@@ -160,12 +161,14 @@ func (c *changeCache) Init(context *DatabaseContext, notifyChange func(base.Set)
 }
 
 // backgroundTask runs task at the specified time interval in its own goroutine until the changeCache is stopped.
-func (c *changeCache) backgroundTask(name string, task func(), interval time.Duration) {
+func (c *changeCache) backgroundTask(name string, task func(ctx context.Context), interval time.Duration) {
 	go func() {
 		for {
 			select {
 			case <-time.After(interval):
-				task()
+				ctx := context.WithValue(context.Background(), base.LogContextKey{},
+					base.LogContext{CorrelationID: base.FormatChangeCacheContextID(c.context.Name)})
```

Comment:
Based on the updated behaviour of the task contexts, I guess we'll need to re-introduce a random portion to the context value.